src/components/PlusButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+type PlusButton = {
+  add: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+export function PlusButton(props: PlusButton) {
+  return (
+    <div>
```

Comment:
unnecessary wrapping div here

Reply:
removed.

Reply:
@peelerzin please don't mark comments as resolved, this should be up to the reviewer

Reply:
@peelerzin also this hasn't been resolved